src/scene/shader-lib/chunks/standard/frag/frontend.js

```diff
@@ -0,0 +1,37 @@
+export default /* glsl */`
+struct Frontend {
+    vec2 uvOffset;
+
+    float alpha;
+
+    vec3 worldNormal;
+
+    vec3 albedo;
+
+    float transmission;
+    float thickness;
+
+    vec3 iridescenceFresnel;
+    float iridescence;
+    float iridescenceThickness;
+
+    vec3 specularity;
+    float glossiness;
```

Comment:
in standard material this was renamed to `gloss` .. should we use the same name here?

Reply:
Yep, done.